AppboyProject/ios/Podfile

```diff
@@ -1,5 +1,20 @@
-
-target 'AppboyProject' do 
+target 'AppboyProject' do
   platform :ios, '8.0'
-  pod 'Appboy-iOS-SDK'
-end
\ No newline at end of file
+
+  # if you want to integrate through Cocoapods instead of manual linking use this:
```

Comment:
Given that our sample project is currently set up to link manually against the React Native bridge, would you mind switching which pods are commented out?

Reply:
Ok, done